src/Caching/SqlServer/src/PublicAPI.Shipped.txt

```diff
@@ -1,4 +1,5 @@
 #nullable enable
```

Comment:
Why aren't the Unshipped.txt files _all_ being emptied in this PR❔

Reply:
I'm not sure, I just ran `./eng/scripts/mark-shipped.cmd` and committed everything (except the submodule change). Can dig in more today.

Reply:
I think in this case it just moved lines around?

Reply:
Yeah, just moving lines around because the script sorts things alphabetically.